presets/4.5/tune/aos_rc/aos_65mm_tune.txt

```diff
@@ -0,0 +1,129 @@
+#$ TITLE: 65mm Tiny whoop tune by Chris Rosser
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: aos, chris, rosser, pid, tune, 65mm, tiny, whoop
+#$ AUTHOR: Chris Rosser
+#$ DISCUSSION: https://www.aos-rc.com
+#$ DESCRIPTION: Custom Tune for a 65mm Tiny whoop. Based on a 300mAh 1S build with 23000KV Motors.
+#$ DESCRIPTION:
+#$ DESCRIPTION: Master multiplier is set to 1.3 for safety. Increase it carefully to further improve flight performance.
+#$ DESCRIPTION:
+#$ DESCRIPTION: RPM filtering is MANDATORY!
+#$ DESCRIPTION:
+#$ DESCRIPTION: Please carefully review the options above and tick any that are relevant.
+#$ DESCRIPTION:
+#$ DESCRIPTION: Choose your RC link speed.
+#$ DESCRIPTION:
+#$ FORCE_OPTIONS_REVIEW: TRUE
+
+#$ INCLUDE: presets/4.5/tune/defaults.txt
+
+# -- PID values --
+set profile_name = AOS 65mm
+set p_pitch = 95
+set i_pitch = 170
+set d_pitch = 44
+set f_pitch = 194
+set p_roll = 76
+set i_roll = 135
+set d_roll = 39
+set f_roll = 156
+set p_yaw = 76
+set i_yaw = 135
+set f_yaw = 156
+set d_min_roll = 39
+set d_min_pitch = 44
+set angle_p_gain = 100
+set simplified_master_multiplier = 130
+set simplified_pi_gain = 130
+set simplified_dmax_gain = 0
+set simplified_pitch_pi_gain = 120
+set tpa_mode = PD
+set tpa_rate = 55
+
+# -- iTerm relax (default)--
+# -- iTerm windup (default)--
+# -- iTerm rotation (off, default) --
+
+# -- Dmax --
+set d_max_advance = 0
+
+# -- TPA --
+
+# -- Feedforward --
+set feedforward_max_rate_limit = 95
+
+# -- PIDsum limits (default)--
+# -- Antigravity (default) --
+# -- Absolute control (off, default) --
+# -- Accecleration limits (off, default) --
+# -- Angle and Horizon mode tuning (default) --
+# -- PIDs active below min throttle (default) --
+# -- Set mixer type to default (legacy) --
+# -- Set yaw spin recovery to default, which is auto --
+# -- Set integrated yaw to off, so pitch and yaw are independent (default) --
+# -- Gyro cal on first arm (off, default) --
+# -- Transient throttle limit (off, default) --
+# -- Thrust linear (off, default) --
+# -- Throttle boost (default, 5)
+# -- VBat warning threshold --
+
+# -- Dynamic idle ON at 5000 rpm --
```

Comment:
dynamic idle can be set up to 25k now... for a 65mm 1s whoop your dynamic idle would be way better off at 15-20k rpm worth trying IMO

Reply:
Raised it to 10k which looks to be a better minimum value. 5k is right on the border with these small motors so there is a risk of desync. I'm a little worried about floatiness at 20kRPM which is around 20% throttle on my test stand.

Reply:
worth a try my preset is  using 20k on the hdzero aio whoop ( mine is modded and 18.9g ) and its not floaty at all.